File: 0001 Code Chef/0016 Minimum Wage/implementation/0004 java/MinimumWage.java
import java.util.Scanner;

class MinimumWage{
    public static void main(String[] args){
        int minimumWage = 11;
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        int inputWage = Integer.parseInt(stringInput);

        if(inputWage > minimumWage){
            System.out.println("YES");
        }else{
            System.out.println("NO");
        }
    }
}

File: 0001 Code Chef/0016 Minimum Wage/implementation/0009 dart/minimum_wage.dart
import 'dart:io';


void main(){
    int minimumWage = 11;
    String? stringInput = stdin.readLineSync();
    if(stringInput != null){
      int? inputWage = int.tryParse(stringInput);
      if(inputWage != null){
        if(inputWage > minimumWage){
          print("YES");
        }else{
          print("NO");
        }
      }
    }
}